Extension/src/packageVersion.ts

```diff
@@ -26,7 +26,13 @@ export class PackageVersion {
             if (firstDigitOffset !== -1) {
                 this.suffix = tokens[3].substring(0, firstDigitOffset);
                 this.suffixVersion = parseInt(tokens[3].substring(firstDigitOffset));
+            } else {
+                this.suffix = tokens[3];
+                this.suffixVersion = 0;
             }
+        } else {
+            this.suffix = tokens[3];
```

Comment:
should suffix be `undefined`?  (I suppose tokens[3] is already undefined in this block, but it seems cleaner to me to just say `undefined` here)

Reply:
Agreed